c/httpfileservice.c

```diff
@@ -68,15 +68,12 @@ bool isDir(char *absolutePath) {
   FileInfo info = {0};
 
   status = fileInfo(absolutePath, &info, &returnCode, &reasonCode);
```

Comment:
I'd keep the check and return ```false```.
```c
if (status == -1) {
  return false;
}
```
The reason is because usually you can't rely on the result of a function if the return value doesn't indicate success. So ```fileInfoIsDirectory``` shouldn't be called in that case.